BankManagerConsoleApp/BankManagerConsoleApp/LinkedList.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  LinkedList.swift
+//  BankManagerConsoleApp
+//
+//  Created by jyubong, Toy on 11/14/23.
+//
+
+final class LinkedList<Data> {
+    private var head: Node<Data>?
+    private var tail: Node<Data>?
+    
+    var first: Data? { head?.data }
+    
+    var isEmpty: Bool { head == nil }
+    
+    init(head: Node<Data>? = nil, tail: Node<Data>? = nil) {
+        self.head = head
+        self.tail = tail
+    }
+    
+    func append(value: Data) {
+        let newNode: Node = Node(data: value)
+        
+        if isEmpty {
+            head = newNode
+            tail = head
+        } else {
+            tail?.setNext(to: newNode)
+            tail = newNode
+        }
```

Comment:
tail에 newNode를 할당하는 작업은 중복이니까 이렇게 변경해도 좋을 것 같아요.

```suggestion
        if isEmpty {
            head = newNode
        } else {
            tail?.setNext(to: newNode)
        }
        
        tail = newNode
```

Reply:
tail에 newNode를 할당하는 작업이 중복된다고 생각되어서 말씀해주신 로직으로 수정하였습니다!